index.js

```diff
@@ -0,0 +1,34 @@
+
+function validatePassword(password) {
+  if (password.length < 8) {
+    return false
+  }
+
+  return hasLowerCaseLetter(password) && hasUpperCaseLetter(password) && hasNumber(password) && hasSymbol(password)
+}
+function hasLowerCaseLetter(password) {
+  var letters = 'abcdefghijklmnopqrstuvwxyz'
+
+  return password.split('').some(element => letters.includes(element))
+}
+
+function hasUpperCaseLetter(password) {
+  var letters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'
+
+  return password.split('').some(element => letters.includes(element))
+}
+
+function hasNumber(password) {
+  var numbers = '0123456789'
+
+  return password.split('').some(element => numbers.includes(element))
+}
+
+function hasSymbol(password) {
+  var symbols = '!@#$%^&*'
+
+  return password.split('').some(element => symbols.includes(element))
```

Comment:
i like this approach that you have. If youre going to split password every time, think about the efficiency of your code. Though I usually encourage breaking up your code into smaller functions. You functions all basically have the same line of code. Ultimately it would have been easier to write one function and one .some() that checked for all of the requirements.